alacritty/src/renderer/platform.rs

```diff
@@ -69,6 +70,24 @@ pub fn pick_gl_config(
         };
 
         if let Some(gl_config) = gl_config {
+            debug!(
+                r#"Picked GL Config:
+  buffer_type: {:?}
+  alpha_size: {}
+  num_samples: {}
+  hardware_accelerated: {:?}
+  supports_transparency: {:?}
+  config_api: {:?}
+  srgb_capable: {}"#,
```

Comment:
I feel like this would be much better like this:

```suggestion
                "Picked GL Config:\n\
                  buffer_type: {:?}\n\
...
```